docs/src/pages/system/styled/styled.md

```diff
@@ -93,3 +93,56 @@ const styled = createStyled({ defaultTheme });
 
 export default styled;
 ```
+
+## Difference with the `sx` prop
+
+We want the `styled` function to behave similarly to the `styled` function of other technologies, like Emotion or Styled Components. 
+The `sx` prop however, is a new way of styling your application, focused on fast customization. `styled` is a function, while `sx` is a prop of Mui components.
+
+Therefore, you may observe a few differences:
+- `sx` provides more shortcuts than `styled`
+- the style definition slightly varies
+- patterns to use props are different
+
+You'll need to take those difference into account when refactoring your components.
+
+### Using props
+
+With `styled`:
+```js
+const MyStyledButton = styled("button")({
+  backgroundColor: props => props.myBackgroundColor
+})
+```
+With `sx`:
+```js
+const MyStyledButton = (props) => (
+  <button sx={{backgroundColor: props.myCustomColor}}>
+     {props.children}
+  </button>
+})
```

Comment:
Maybe it's worth sharing that if you use callback for some `sx` prop, it is the `theme` you are receiving as a param. For example:

```
  <button sx={{ background: theme => theme.palette.primary.main }}>
     {props.children}
  </button>
```